data-science-onramp/feature-engineering/citibike.ipynb

```diff
@@ -0,0 +1,385 @@
+    "\n",
+    "+ [Cloud SDK](https://cloud.google.com/sdk/) is a command line tool which allows you to interact with Google Cloud products. This notebook introduces several `gcloud` and `gsutil` commands, which are part of the Cloud SDK. Note that shell commands in a notebook must be prepended with a `!`.\n",
+    "\n",
+    "+ [Pandas](https://pandas.pydata.org/) is a data analysis and manipulation tool built on top of the Python programming language."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set up your environment\n",
+    "\n",
+    "### Enable the required APIs\n",
+    "\n",
+    "In order to use AI Platform, confirm that the required API is enabled:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud services enable ml.googleapis.com"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load the data\n",
+    "\n",
+    "## Import libraries\n",
+    "Running the following cell will import the libraries needed. \n",
+    "\n",
+    "+ `Pandas`: to store and manipulate the dataset.\n",
+    "* `Google Cloud Storage`: to retrieve the dataset from the GCS bucket where the dataset is stored."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from google.cloud import storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define constants\n",
+    "Define the name of your Google Cloud Storage bucket where the cleaned data is stored. \n",
+    "\n",
+    "+ `BLOB_PREFIX`: indicates the folder where the files are stored.\n",
+    "+ `DIR_NAME`: holds the name of the local folder where the files will be downloaded to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BUCKET_NAME = 'your-bucket-name'\n",
+    "BLOB_PREFIX = 'clean_citibike_data.csv.gz/part'\n",
+    "DIR_NAME = 'citibike_data'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the data\n",
+    "\n",
+    "Run the following command to create a local folder where the dataset files will be stored."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir $DIR_NAME"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following cell will retrieve all of the files with the `BLOB_PREFIX` defined above and download them into the folder. You will also create a list of the file names so they can be referenced later when loading the data into a dataframe."
```

Comment:
Nit -> instead of "you will also" you should probably say "It will also" because the cell is doing this - the user doesn't have to